src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -153,6 +153,11 @@ public function storeHasAConfigurableProduct($productName)
         $product->setCode($this->convertToCode($productName));
         $product->setDescription('Awesome '.$productName);
 
+        if ($this->sharedStorage->has('channel')) {
+            $channel = $this->sharedStorage->get('channel');
```

Comment:
`$channel` is not used anywhere else. ` $product->addChannel($this->sharedStorage->get('channel'));`